src/holofuel/components/Header/Header.js

```diff
@@ -20,9 +21,15 @@ export function Header ({ title, agent, agentLoading, avatarUrl, history: { push
         {leftNav}
         <span styleName='title header-font'>HoloFuel</span>
       </div>
-      <div styleName='right-nav account-number header-font'>{agent.nickname || presentAgentId(agent.id)}</div>
+      <div styleName='right-nav account-number header-font'>
+        <CopyAgentId agent={agent} isMe>
+          {agent.nickname || presentAgentId(agent.id)}
+        </CopyAgentId>
+      </div>
       <Link to='/history' styleName='avatar-link'>
-        <HashAvatar avatarUrl={avatarUrl} seed={agent.id} size={32} />
+        <CopyAgentId agent={agent} isMe>
```

Comment:
So this `CopyAgentId` is wrapped inside a `Link` to `'/history'`. When you click do both happen? That's probably not what we want. This is a UX question. Let's remove the `Link` for now and ask C and Z for clarification.